store/postgres/src/notification_listener.rs

```diff
@@ -25,6 +26,15 @@ lazy_static! {
                 )))
             )
             .unwrap_or(Duration::from_secs(300));
+    static ref DB_NOTIFICATION_BROADCAST_TIMEOUT: Duration =
+        env::var("GRAPH_DB_NOTIFICATION_BROADCAST_TIMEOUT")
```

Comment:
Can we just call this `GRAPH_NOTIFICATION_BROADCAST_TIMEOUT`?

Reply:
Done